style.css

```diff
@@ -1079,3 +1078,154 @@ header {
   left: -35px;
   top: 90px;
 }
+
+.contact-img2 {
+  position: relative;
+  width: 231px;
+  height: 708px;
+  left: 176px;
+  top: -727px;
+}
+
+.contact-img {
+  position: relative;
+  width: 396px;
+  height: 781px;
+  left: 11px;
+  top: 11px;
+}
+
+.contact p {
+  position: relative;
+  width: 327px;
+  height: 102px;
+  left: 48px;
+  top: -777px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact h2 {
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 0;
+  margin-top: -582px;
+  top: -802px;
+  left: 89px;
+  color: #fff;
+}
+
+.contact {
+  position: relative;
+  width: 480px;
+  height: 920px;
+  left: 0;
+  top: 1600px;
+}
+
+.user-name ::placeholder {
+  position: absolute;
+  left: 16px;
+  right: 38px;
+  top: 10px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.user-email ::placeholder {
+  position: absolute;
+  left: 16px;
+  right: 38px;
+  top: 10px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+textarea {
+  position: absolute;
+  left: 42px;
+  top: 480px;
+  bottom: 10px;
+  width: 320px;
+  height: 170px;
+  border-radius: 8px;
+  margin-top: 20px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  color: #b3bac5;
+}
+
+input {
+  position: relative;
+  font-family: "Poppins", sans-serif;
```

Comment:
- [ ] Please remove the outline of your inputs to match the figma design: 
![image](https://user-images.githubusercontent.com/91486791/220362806-d0540bd4-44d5-4e8a-a92f-bb62365db638.png)
![image](https://user-images.githubusercontent.com/91486791/220362869-1987203d-4a28-4bc4-8eff-0031a6b824a2.png)